parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -1128,4 +1128,32 @@ EOF
     echo "Enable localdns succeeded."
 }
 
+startNvidiaDCGMExporterService() {
+    # Start the nvidia-dcgm service.
+    systemctlEnableAndStart nvidia-dcgm 30 || exit $ERR_NVIDIA_DCGM_FAIL
+
+    # Create systemd drop-in directory for nvidia-dcgm-exporter service
+    DCGM_EXPORTER_OVERRIDE_DIR="/etc/systemd/system/nvidia-dcgm-exporter.service.d"
+    mkdir -p "${DCGM_EXPORTER_OVERRIDE_DIR}"
+
+    # Create drop-in file to override service configuration
+    DCGM_EXPORTER_OVERRIDE_FILE="${DCGM_EXPORTER_OVERRIDE_DIR}/10-aks-override.conf"
+
+    tee "${DCGM_EXPORTER_OVERRIDE_FILE}" > /dev/null <<EOF
+[Service]
+# Remove file-based logging - let systemd handle logs
+StandardOutput=journal
```

Comment:
do we have a way to extract the logs to a file when we generate the CSE Archive (GetVmAGentLog)  normally we grab everything under /var/log/azure , if there is no log files there we might what to augment the log dumping logic to run a journalctl command to export it.

Reply:
Where should this log collection happen?